src/miner_lora.erl

```diff
@@ -150,14 +151,37 @@ location_ok() ->
     %% this terrible thing is to fake out dialyzer
     application:get_env(miner, loc_ok_default, true).
 
--spec reg_domain_data_for_addr(libp2p_crypto:pubkey_bin())-> {error, any()} | {ok, freq_data()}.
-reg_domain_data_for_addr(Addr)->
-    case country_code_for_addr(Addr) of
-        {ok, CC} ->
-            %% use country code to get regulatory domain data
-            ?MODULE:reg_domain_data_for_countrycode(CC);
-        {error, Reason} ->
-            {error, Reason}
+-spec reg_domain_data_for_addr(libp2p_crypto:pubkey_bin(), state())-> {error, any()} | {ok, freq_data()}.
+reg_domain_data_for_addr(Addr, #state{chain=Chain}) ->
+    case blockchain:ledger(Chain) of
```

Comment:
This doesn't check for an undefined chain.

Reply:
Updated in  7809680